### PR TITLE
Export rml mappings

### PR DESCRIPTION
**Pull request by wmaroy**
Added rml export functionality, relevant commits are from Jun 09, 2016.

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -132,6 +132,11 @@
             <artifactId>jackson-module-scala_2.11</artifactId>
             <version>2.5.0</version>
         </dependency>
+        <dependency>
```
Comment (jimkont) [inline core/pom.xml]: I think this causes the build error

File: server/server.default.properties [hunk 2]
```diff
@@ -16,7 +16,7 @@ ontologyFile=../ontology.xml
 mappingsDir=../mappings
 
 # List of languages, e.g. 'en,de,fr' or '@mappings'
-languages=@mappings
```
Comment (jimkont) [inline server/server.default.properties]: this shouldn't be in the commit

File: server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package org.dbpedia.extraction.server.resources.rml.model
+
+/**
+  * Contains the prefixes map
+  */
+object Prefixes {
```
Comment (jimkont) [inline server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala]: we should reuse the existing ns code from here
https://github.com/dbpedia/extraction-framework/blob/398bd582b3d2233531c8f3cb94270208392b0910/core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala#L30-L30

**Comment by wmaroy** [comment]
It compiles now, I also updated the code to reuse the ns code

File: core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala [hunk 4]
```diff
@@ -65,6 +65,14 @@ object RdfNamespace {
   val MAPPINGS = ns("mappings", "http://mappings.dbpedia.org/wiki/", false)
   val D0 = ns("d0", "http://www.ontologydesignpatterns.org/ont/d0.owl#", false)
   val DUL = ns("dul", "http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#", false)
+  val RR = ns("rr", "http://www.w3.org/1999/02/22-rdf-syntax-ns#")
+  val RML = ns("rml", "http://semweb.mmlab.be/ns/rml#")
+  val QL = ns("ql", "http://semweb.mmlab.be/ns/ql#")
+  val DBO = ns("dbo", "http://dbpedia.org/ontology/")
+  val FNML = ns("fnml", "http://semweb.mmlab.be/ns/fnml#")
+  val FNO = ns("fno", "http://w3id.org/function/ontology/")
+  val EX = ns("ex", "http://example.org/stuff/1.0/")
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala]: do we need this one? :)

File: server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala (hunk 3 above)
Comment (jimkont) [inline server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala]: Do we still need this class?

File: core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala (hunk 4 above)
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala]: I meant this do we need this file now? looks like all references to it are deleted

File: server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala (hunk 3 above)
Comment (wmaroy) [inline server/src/main/scala/org/dbpedia/extraction/server/resources/rml/model/Prefixes.scala]: No indeed, I replaced the usage with the RdfNamespace class now, I'll delete it

File: core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala (hunk 4 above)
Comment (wmaroy) [inline core/src/main/scala/org/dbpedia/extraction/ontology/RdfNamespace.scala]: The prefixes are used for generating the RML mappings from the DBpedia mappings, I'll delete the EX prefix

**Pull request merged**